Components/Categories.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export default function Categories() {
+  return (
+    <>
+      <CatBody>
+        <h1>Kategorien</h1>
+        <a>Kind</a>
+        <a>Auto</a>
+        <a>Haus</a>
+        <a>Mode</a>
+        <a>Beauty</a>
+        <a>Haustiere</a>
+        <a>Elektronik</a>
+        <a>Musik</a>
+        <a>Filme</a>
```

Comment:
Will this become the navigation at a later point? If so, then I would consider wrapping the `a`-tags inside a `nav` element